src/avatar/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import cn from 'classnames';
+import img from './missing.png';
+
+import styles from './styles.css';
+
+export default ({ src = '', inline = '', defaultImage = img }) => {
+  const image = src || defaultImage;
```

Comment:
oh it's so you can override the default image, duh. Um, i dunno, if we don't actually do that anywhere you won't be breaking the api. I'm not sure if that's a useful part of the api to keep.